docs/domains/Domain.json

```diff
@@ -297,6 +302,9 @@
           "type": "number",
           "minimum": 0
         },
+        "modified": {
```

Comment:
We shouldn't add this field.  The "modified" field in the processing logic was to just to track when the operator has a pending change to the status.

Reply:
Now ignoring volatile fields in schema generation